### ISSUE TITLE
Server side pdfmake embeds fonts incompletely

### ISSUE THREAD

**Issue by ValeSauer** [bpampuch/pdfmake#1659]
I was searching for answers all over github and stackexchange and could not find a working solution for this issue. 

I try to embed font [FiraSans](https://fonts.google.com/specimen/Fira+Sans). However, it seems to embedded incompletely. Only a view characters are actually displayed correctly. I can copy even the "invisible" characters to any text editor and I receive the correct text.

I tried three approaches:
1. The one from the [examples](https://github.com/bpampuch/pdfmake/blob/0.1/examples/basics.js). However I think this is a bad example because it uses Roboto, which seems to be included anyway and works.
2. The one from the [server-side dev-playground](https://github.com/bpampuch/pdfmake/blob/master/dev-playground/server.js). I thought this might solve path issues because I am working a windows platform, but it didn't work.
3. The client side vs_fonts.js approach which is propoped in the [github issues](https://github.com/bpampuch/pdfmake/issues/1312). 

None of them worked for me. They actually all create the same result, which looks like that:
![image](https://user-images.githubusercontent.com/7767250/53650979-07568a00-3c46-11e9-96eb-9f02dfb2080d.png) 

I can exclude the following causes:
- incorrect path, it would throw errors. already tried out.
- corrupted ttf files, as i tried it with various fonts and sources

Does anyone have a solution for this? 

My code:
```javascript
const pdfMakePrinter = require('pdfmake/src/printer');
const fs = require('fs')
var path = require('path');

const docDefinition = {
    content: [
        'abcdefghijklmnopqrstuvwABCDEFGHIJKLMNOPQRSTUVW0123456789',
    ],
    defaultStyle: {
        font: 'FiraSans'
    }
};

var generatePdf = function (docDefinition, callback) {
    try {
        const fontDescriptors = {
            FiraSans: {
                normal: './fonts/FiraSans-Regular.ttf',
                //normal: path.join(__dirname, '..', 'pdf-generate', '/fonts/FiraSans-Regular.ttf'),
                //normal: new Buffer(require('pdfmake/build/vfs_fonts.js').pdfMake.vfs['FiraSans-Regular.ttf'], 'base64'),
                bold: '/fonts/FiraSans-Medium.ttf',
                italics: '/fonts/FiraSans-Italic.ttf',
                bolditalics: '/fonts/FiraSans-MediumItalic.ttf'
            }
        };
        const printer = new pdfMakePrinter(fontDescriptors);
        const doc = printer.createPdfKitDocument(docDefinition);

        doc.pipe(
            fs.createWriteStream('docs/filename.pdf').on("error", (err) => {
                console.error(err.message);
            })
        );
        doc.on('end', () => {
            console.log("PDF successfully created and stored");
        });
        doc.end();

    } catch (err) {
        throw (err);
    }
};

generatePdf(docDefinition, (response) => {
    res.setHeader('Content-Type', 'application/pdf');
    res.send(response); // Buffer data
});
```

**Comment by blikblum** [comment]
For the record. 

Loading FiraSans in https://fontkit-demo.now.sh/ works.

But in pdfkit not: https://repl.it/@blikblum/pdfkit-firasans

**Comment by ValeSauer** [comment]
This PR  solves the issue in way, that the server side embedding works with the code below. There are no other workarounds or anything needed. 
```javascript
FiraSans: {
   normal: './fonts/FiraSans/FiraSans-Regular.ttf',
   bold: './fonts/FiraSans/FiraSans-Medium.ttf',
   italics: './fonts/FiraSans/FiraSans-Italic.ttf',
   bolditalics: './fonts/FiraSans/FiraSans-MediumItalic.ttf'
 },
```

**Comment by liborm85** [comment]
Released in 0.1.54.

**Comment by blunatic** [comment]
This still seems to be an issue on the client side use of fonts even with 0.1.54. 

I've been successfully using a custom font (Proxima Nova) for awhile now, but when I try to switch to a different custom font (Source Sans Pro) embedded as a base64 string I get the following:

![image](https://user-images.githubusercontent.com/5156743/56307115-fa5c0000-60f8-11e9-9d9a-921b04b05766.png)
